src/sdk/api/constants.ts

```diff
@@ -18,6 +15,7 @@ export const API_ENDPOINTS = {
   GET_ACCOUNT_BALANCES: 'account/balances',
   GET_ACCOUNT_NFTS: 'account/nfts',
   GET_TRANSACTION: 'transactions/transaction',
```

Comment:
This is pretty misleading? What does transactions/transaction mean? Can't we change this to something more meaningful

Reply:
The route has been modified to `transactions/transactionByHash`